pro/src/pages/IndividualOffers/IndividualOffersContainer/components/IndividualOffersTable/components/IndividualOfferRow/components/OfferBookingCell/OfferBookingCell.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames'
+import { isAfter } from 'date-fns'
+
+import { ListOffersOfferResponseModel, OfferStatus } from 'apiClient/v1'
+import { useActiveFeature } from 'commons/hooks/useActiveFeature'
+import { FORMAT_DD_MM_YYYY_HH_mm } from 'commons/utils/date'
+import { formatLocalTimeDateString } from 'commons/utils/timezone'
+import { getDepartmentCode } from 'components/IndividualOffer/utils/getDepartmentCode'
+import { getCellsDefinition } from 'components/OffersTable/utils/cellDefinitions'
+import { Tag, TagVariant } from 'design-system/Tag/Tag'
+import waitFullIcon from 'icons/full-wait.svg'
+import styles from 'styles/components/Cells.module.scss'
+
+export type OfferBookingCellProps = {
+  rowId: string
+  offer: ListOffersOfferResponseModel
+  className?: string
+}
+
+export const OfferBookingCell = ({
+  rowId,
+  offer,
+  className,
+}: OfferBookingCellProps) => {
+  const isRefactoFutureOfferEnabled = useActiveFeature(
+    'WIP_REFACTO_FUTURE_OFFER'
+  )
+
+  const departmentCode = getDepartmentCode(offer)
+
+  const bookableDate =
+    [OfferStatus.SCHEDULED, OfferStatus.PUBLISHED].includes(offer.status) &&
+    offer.bookingAllowedDatetime &&
+    isAfter(offer.bookingAllowedDatetime, new Date())
+      ? formatLocalTimeDateString(
+          offer.bookingAllowedDatetime,
+          departmentCode,
+          FORMAT_DD_MM_YYYY_HH_mm
+        )
+      : null
+
+  return (
+    <td
+      role="cell"
+      className={classNames(
+        className,
+        styles['offers-table-cell'],
+        styles['bookings-column']
+      )}
+      headers={`${rowId} ${getCellsDefinition(isRefactoFutureOfferEnabled).INDIVIDUAL_BOOKINGS.id}`}
+    >
+      <span
+        className={styles['offers-table-cell-mobile-label']}
+        aria-hidden={true}
+      >
+        {`${getCellsDefinition(isRefactoFutureOfferEnabled).INDIVIDUAL_BOOKINGS.title} :`}
+      </span>
+      {bookableDate ? (
+        <span style={{ whiteSpace: 'nowrap' }}>
+          <Tag
+            label={bookableDate}
+            icon={waitFullIcon}
+            variant={TagVariant.WARNING}
+          />
+        </span>
+      ) : (
+        offer.bookingsCount || '-'
```

Comment:
```suggestion
        offer.bookingsCount ?? '-'
```

Plus safe ? Je propose d'ailleurs d'ajouter ça au linter si ça te paraît aussi plus safe ;).

Reply:
J'ai juste déplacer le code, je ne préfère pas toucher